test/parallel/test-inspector-multisession-ws.js

```diff
@@ -0,0 +1,75 @@
+// Flags: --expose-internals
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+
+const { NodeInstance } = require('../common/inspector-helper.js');
+
+// Sets up JS bindings session and runs till the sessions gets "paused" event
+const script = `
+const { Session } = require('inspector');
+const session = new Session();
+let done = false;
+const interval = setInterval(() => {
+  // Keep it running
+}, 150);
+session.on('Debugger.paused', () => {
+  session.disconnect();
+  process.exit(0);
```

Comment:
It would be infinitesimally nicer to have node exit naturally by calling `clearInterval(interval)`.

Reply:
Done